Solutions/Milyli.ScriptRunner.Services/V1/ScriptManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Milyli.ScriptRunner.Services.V1
```

Comment:
We can leave this project off for now since it is just NotImplementedExceptions.

- [ ] Remove placeholder Services implementation project